keep/api/core/db.py

```diff
@@ -2107,13 +2111,16 @@ def get_last_incidents(
             session.query(
                 Incident,
             )
-            # .options(joinedload(Incident.alerts))
+            .options(joinedload(Incident.alerts))
```

Comment:
?

Reply:
Someone's recent commit introduced this line as commented. I need joinedload uncommented.

If this is a necessary optimization, I can introduce a flag to function params

Reply:
who introduced it and why? if he fixed a bug you don't know about - you are just adding this bug again

Reply:
Introduced by @VladimirFilonov [here](https://github.com/keephq/keep/commit/51905691d6f576f10e1605dca16cd52558467f6d). According to our discussion, may be uncommented for now.